app/uk/gov/hmrc/gform/sdes/SdesConnector.scala

```diff
@@ -38,5 +38,11 @@ class SdesConnector(wSHttp: WSHttp, sdesBaseUrl: String, sdesBasePath: String, h
     wSHttp
       .POST[SdesNotifyRequest, HttpResponse](url, payload, headers)
       .failWithNonSuccessStatusCodes(url)
+      .recoverWith { case e =>
+        logger.error(
+          s"Failed to notify SDES [file-ready] for the information type ${payload.informationType}, error : $e"
+        )
+        Future.failed(e)
+      }
```

Comment:
I think this is not right place to log. You are ignoring the exception [here](https://github.com/hmrc/gform/blob/main/app/uk/gov/hmrc/gform/scheduler/QueueAlgebra.scala#L36) and that is the place where we should be logging the error